packages/coinstac-api-server/src/auth-helpers.js

```diff
@@ -10,14 +10,24 @@ const { eventEmitter, USER_CHANGED } = require('./data/events');
 
 sgMail.setApiKey(process.env.SENDGRID_API_KEY);
 
+const audience = 'coinstac';
```

Comment:
maybe set this via env variables?

Reply:
agreed, but we need to figure out what the best practice is here, I think this is just to get this off the ground for now